neso_solar_consumer/app.py

```diff
@@ -3,31 +3,45 @@
 
 This script orchestrates the following steps:
 1. Fetches solar forecast data using the `fetch_data` function from `fetch_data.py`.
-2. Formats the forecast data (currently a placeholder for future enhancements).
-3. Saves the forecast data (currently a placeholder for future enhancements).
+2. Formats the forecast data into `ForecastSQL` objects using `format_forecast.py`.
+3. Saves the formatted forecasts into the database using `save_forecast.py`.
 
 Functions:
     - get_forecast: Fetches raw solar forecast data.
-    - format_forecast: Processes the forecast data for a specific format (placeholder for now).
-    - save_forecast: Saves the formatted forecast data (placeholder for now).
+    - format_forecast: Converts raw forecast data into `ForecastSQL` objects.
+    - save_forecast: Saves the formatted forecast data to the database.
 """
 
-from .fetch_data import fetch_data
+import logging
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+from neso_solar_consumer.fetch_data import fetch_data
+from neso_solar_consumer.format_forecast import format_to_forecast_sql
+from neso_solar_consumer.save_forecast import save_forecasts_to_db
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s [%(levelname)s] %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+# Database configuration (replace with actual database URL)
```

Comment:
you ok to remove this, we should get a sessions, using something like
https://github.com/openclimatefix/analysis-dashboard/blob/main/src/pvsite_forecast.py#L93